crates/stages/src/stages/merkle.rs

```diff
@@ -35,33 +37,81 @@ pub const MERKLE_UNWIND: StageId = StageId("MerkleUnwind");
 #[derive(Debug)]
 pub enum MerkleStage {
     /// The execution portion of the hashing stage.
-    Execution,
+    Execution {
```

Comment:
You mean something like this?
```rust
impl Default for MerkleStage {
    fn default() -> Self {
        Self::Execution { clean_threshold: 1000 }
    }
}
```
Wouldn't that be a bit confusing, since we also have the `Unwind` variant? I was thinking of having a `default_<variant>()` for each one, maybe that'd be more explicit. wdyt?